code/modules/mob/living/basic/space_fauna/cat_surgeon.dm

```diff
@@ -54,6 +54,9 @@
 	if(QDELETED(tail))
 		return
 
+	if(tail.icon_state == "none")
+		return // I hate pref organ code
```

Comment:
In a perfect world, invisible tails would not get added in the first place.... but this isn't a perfect world.

Reply:
What the fuck